sqlglot/dialects/mysql.py

```diff
@@ -27,12 +44,37 @@ class MySQL(Dialect):
     class Parser(Parser):
         STRICT_CAST = False
 
+        def _parse_ordered(self):
+            is_null = self._match(TokenType.ISNULL)
+            if is_null:
+                self._match(TokenType.L_PAREN)
+                this = self._parse_conjunction()
```

Comment:
why do you do this manually?

should you just parse a conjunction , allow ISNULL to be a function and use that?